### DISCUSSION TITLE
Am I doing it wrong?

### DISCUSSION THREAD

**Discussion by tismofied** [moralmunky/Home-Assistant-Mail-And-Packages#524] [Q&A] [answered]
Please bare with me here. I am not good with coding. I am on HA 2021.10.1 and latest mail and packages integration.I followed the installation process step by step but I feel I didn't something in the steps or missed it because I can never get packages either from usps or Amazon to show in the Lovelace card. The only thing working is usps mail. I included pics of my integration config. Please let me know if I am doing it wrong. Thank you!
![image](https://user-images.githubusercontent.com/90809124/136655751-fb17699a-96db-4cbe-b043-de5a5b6fe21f.jpeg)

![image](https://user-images.githubusercontent.com/90809124/136655757-c3a8976e-37de-4002-b505-571157f6ca1c.jpeg)

![image](https://user-images.githubusercontent.com/90809124/136656112-d649d474-7421-40f3-bb4c-0472872c7280.jpeg)

Also getting these errors. Have no idea what they mean.

![image](https://user-images.githubusercontent.com/90809124/136656131-115742ad-2189-4e28-a5f4-14d1f5f811c9.jpeg)

![image](https://user-images.githubusercontent.com/90809124/136656133-db4ebf0d-a36f-44fc-ad9c-820caeef5137.jpeg)

**Comment by firstof9** [comment]
What version of the integration is this?

**Reply by tismofied** [reply]
0.06

**Reply by firstof9** [reply]
No that's the version of the card.

**Reply by tismofied** [reply]
sorry. its  0.3.2

**Reply by firstof9** [reply]
Try the latest beta.

In HACS, select the `Reinstall` option via the overflow menu (the 3 dots) as shown:
![image](https://user-images.githubusercontent.com/1105672/136676062-8e47f9ce-69e5-4e79-b2e8-ac36dceaa83a.png)


Then enable Betas as shown:
![image](https://user-images.githubusercontent.com/1105672/136676074-9565cb88-03ff-40a3-86ca-e8fb0260b468.png)

Then select the 0.3.3-b20 and hit `Install`

**Reply by tismofied** [reply]
Thank you for your help. I did as you said. Now do I just have to wait for a day where a package is being  delivered to see if it works or not?

**Reply by firstof9** [reply]
If you are no longer getting errors, then it's fixed, but yes to confirm everything's working as intended for you, you'd want to wait for a day a package is being delivered.

**Reply by tismofied** [reply]
![image](https://user-images.githubusercontent.com/90809124/136851075-8d566951-250c-4551-b66f-79d072c79dbb.png)

🙌 thank you so much!!!

**Reply by tismofied** [reply]
Today I am getting 5 mail and I noticed the gif sensor is blank. 
Anyway I can fix this?![image](https://user-images.githubusercontent.com/90809124/136972614-1401f9ae-e630-416f-a8cc-f201b1e202ee.png)

**Comment by firstof9** [comment, marked as answer]
Make sure it's set to the `camera.mail_usps_mail` entity? I don't use the card so I'm not 100% sure.

**Reply by tismofied** [reply]
I tried that but it borked the card lol. I'll ask in the card thread on the HA community. Hopefully someone have an answer there. 

One more thing please, how can I get the message to show in the card. I understand I need a template but I have no clue how to set one up

**Reply by firstof9** [reply]
[This wiki link](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Mail-Summary-Message#text-summary-examples) has the information you need for a template sensor.
